backend/src/emissions/schema.py

```diff
@@ -812,19 +814,17 @@ class Arguments:
 
         input = ElectricityInput(required=True)
 
-    ok = graphene.Boolean()
+    success = graphene.Boolean()
     electricity = graphene.Field(ElectricityType)
 
     @staticmethod
     @login_required
+    @representative_required
     def mutate(root, info, input=None):
         """Process incoming data"""
         user = info.context.user
-        ok = True
-        if not user.is_representative:
-            raise GraphQLError(
```

Comment:
Why don't we need this error anymore?

Reply:
Ah ja good question. I just looked at it again. It's because the decorator `@representative_required` already checks that the user is a representative.